kalite/main/api_views.py

```diff
@@ -57,104 +40,6 @@ def student_log_api_wrapper_fn(request, *args, **kwargs):
         return student_log_api_wrapper_fn
 
 
-@student_log_api(logged_out_message=ugettext_lazy("Video progress not saved."))
```

Comment:
Except save_video_log for some reason. Having burned it down, I will recreate it in tastypie now...